### PR TITLE
Create rules page

### PR DESCRIPTION
**Pull request by BlueChinchompa**

### CODE REVIEW

File: wiki/Rules/en.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Rules
+
+These are the rules that help keep the game and community fun and fair.
+
+# Community Rules
+
+1. **Each user may have only one account EVER!** The only exception to this rule is making bot accounts or other projects, but you must first let staff know so we can approve it. If you have lost your email or password please open a support ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc).
```
Comment (iSlodinxOsu) [inline wiki/Rules/en.md:7]:
```suggestion
1. **Each user may have only *ONE* account ever.** The only exception to this rule is making bot accounts or other projects, but you must first let staff know so we can approve it. If you cannot access your email or you forgot the password, please open a support ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc).
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/Rules/en.md (hunk 1 above)
Comment (iSlodinxOsu) [reply wiki/Rules/en.md:7]: oh apologize, put the caps on "EVER"

File: wiki/Rules/en.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+2. **Play fair.** Using any third party tools that give you an unfair advantage over people will result in a restriction. This includes programs like macros, cheat engine, aim-assist programs, timewarp, and so on. If a program is doing something to help you play that you should be doing then it is not okay! If you catch anybody cheating please open up a ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc) server.
+3. **Be kind to people.** Any harrassment, racism, or antagonism has no place here in Titanic! This is just a server about clicking circles, catching fruit, bashing drums, and breaking keyboards whilst pretending to be cookiezi and playing old maps, not to be assholes to each other.
```
Comment (iSlodinxOsu) [inline wiki/Rules/en.md:9]:
```suggestion
3. **Be kind to people.** Any harrassment, racism, or antagonism has no place here in Titanic! This server is about clicking circles, catching fruit, bashing drums, and breaking keyboards whilst pretending to be Cookiezi and playing old maps, not to be assholes to each other. 
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

**Review by iSlodinxOsu — APPROVED** [review]

File: wiki/Rules/en.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Rules
+
+These are the rules that help keep the game and community fun and fair.
+
+# Community Rules
+
+1. **Each user may have only *one* account EVER!** The only exception to this rule is making bot accounts or other projects, but you must first let staff know so we can approve it. If you cannot access your email or you have forgot your password please open a support ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc).
```
Comment (iSlodinxOsu) [inline wiki/Rules/en.md:7]:
```suggestion
1. **Each user may only *one* account EVER!** The only exception to this rule is making bot accounts or other projects, but you must first let staff know so we can approve it. If you cannot access your email or you have forgot your password please open a support ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc).
```

Comment (iSlodinxOsu) [inline wiki/Rules/en.md:7]:
```suggestion
1. **Each user may have only *one* account EVER!** The only exception to this rule is making bot accounts or other projects, but you must let staff know so we can approve it. If you cannot access your email or you have forgot your password please open a support ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc).
```

File: wiki/Rules/en.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+2. **Play fair.** Using any third party tools that gives you an unfair advantage over people will result in a restriction. This includes programs like macros, cheat engine, aim-assist programs, timewarp, and so on. If a program is doing something to help you play that you should be doing then it is not okay! If you see some suspicious activity from the user, please open up a ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc) server.
+3. **Be kind to people.** Any harrassment, racism, or antagonism has no place here in Titanic! This is just a server about clicking circles, catching fruit, bashing drums, and breaking keyboards whilst pretending to be cookiezi and playing old maps, not to be assholes to each other.
+4. **We are an all-ages community.** This means that any 18+/NSFW content such as drug use or topics of sexual nature are not welcome here.
```
Comment (Lekuruu) [inline wiki/Rules/en.md:10]: I would not say that we are an "all-ages" community. The minimum age is 13+ just to be safe here.

**Comment by TheQuickBrownFoxLF** [comment]
suggestion: the “rules” tab should say something along the lines of “Welcome to titanic. Before you get started please read through these rules for a more safe and pleasant experience” or something similar. As the current text seems waaay to blunt

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]
seems like i forgot to double-check it but here ya go

File: wiki/Rules/en.md (hunk 4 above)
Comment (iSlodinxOsu) [reply wiki/Rules/en.md:10]: yeah i actually agree.

Comment (BlueChinchompa) [reply wiki/Rules/en.md:10]: I guess making it something on the lines of, "No NSFW/18+ Content allowed" instead of all-ages.

**Comment by Digitalfear117** [comment]
`If a program is doing something to help you play that you should be doing then it is not okay! If you see some suspicious activity from the user, please open up a ticket in the [Titanic discord](https://discord.gg/qryYG2C5nc) server.`

That first sentence doesn’t make any sense, and honestly is a bit redundant anyways. Maybe just axe it.

**Review by iSlodinxOsu — APPROVED** [review]

**Review by Lekuruu — APPROVED** [review]
I would suggest to add a "What happens if you break the rules" section, but we could also do that in another pr.

**Pull request merged**